frontend/src/layout/navigation-3000/navigationLogic.ts

```diff
@@ -202,10 +267,39 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
             },
         ],
         sidebarContentsFlattened: [
-            (s) => [(state) => s.activeNavbarItem(state)?.logic.findMounted()?.selectors.contents(state)],
```

Comment:
The following "silent" error in console seems likely related to this:

<img width="975" alt="Screenshot 2023-07-18 at 10 24 42" src="https://github.com/PostHog/posthog/assets/1851359/d907f393-7ce8-49b2-bfb1-097851b94bcb">

Reply:
Fixed